pyproject.toml

```
@@ -9,7 +9,7 @@ repository = "https://github.com/j-martin/sphinx-diagrams"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-diagrams = "^0.17.0"
+diagrams = "^0.20.0"
```

Let's do this instead

```suggestion
diagrams = "0.*"
```